app/uk/gov/hmrc/accessibilitystatementfrontend/models/AccessibilityStatement.scala

```diff
@@ -30,14 +31,16 @@ case class AccessibilityStatement(
   complianceStatus: ComplianceStatus,
   accessibilityProblems: Option[Seq[String]],
   milestones: Option[Seq[Milestone]],
-  automatedTestingOnly: Option[Boolean],
+  automatedTestingOnlyOption: Option[Boolean],
```

Comment:
@kyle-bowden This change won't work as the parser expects the case class property name to match the JSON (from YAML) field name. You can view by running the branch locally and comparing:

http://localhost:12346/accessibility-statement/example-automated-testing-only
https://www.staging.tax.service.gov.uk/accessibility-statement/example-automated-testing-only

So we'll need to revert this name back.